shell_completions.md

```diff
@@ -431,4 +439,38 @@ search  show  status
 
 ## PowerShell completions
 
-Please refer to [PowerShell Completions](powershell_completions.md) for details.
+Cobra supports native PowerShell completions generated from the root `cobra.Command`. You can use the `command.GenPowerShellCompletion()` or `command.GenPowerShellCompletionFile()` functions. You must provide these functions with a parameter indicating if the completions should be annotated with a description. Cobra will provide the description automatically based on usage information. You can choose to make this option configurable by your users.
```

Comment:
This needs to be update to follow the API that will be chosen (which cannot be this one).